src/combat.ts

```diff
@@ -378,8 +378,8 @@ export class Macro extends StrictMacro {
       .kill();
   }
 
-  static meatKill(): Macro {
-    return new Macro().meatKill();
+  static meatKill(turnCount: number): Macro {
```

Comment:
This doesn't have to be in scope but it is probably worth separating out our embezzler macro from our barf macro (romance wanderers mean we do still need embezzler code in the meatkill code, but still), because it seems tedious to be passing a turncount to `meatKill` every dingdang time when it almost never matters.

Reply:
yeah okay it absolutely sucks ass to have to pass `turnCount` into a billion different functions because of this change; make it an optional parameter and make it being defined part of the condition used to redigitize, I think.

Reply:
(note: when sweaty bill noted that this sucks ass, he did not mean the way you did this inherently sucks ass, just that it sucked that you had to do this due to the interconnectivity of garbo's systems lol)

Reply:
I didn't take it negatively. No worries. 

There's a a few functions that we call multiple times and I wonder if we can call it once and store the result. `estimatedGarboTurns` is one of them.

Reply:
done!